tests/TestCase/Database/ConnectionTest.php

```diff
@@ -962,4 +962,27 @@ public function testSchemaCollection()
         $connection->schemaCollection($schema);
         $this->assertSame($schema, $connection->schemaCollection());
     }
+
+    /**
+     * Tests supportsCrossWith
+     *
+     * @return void
+     */
+    public function testSupportsCrossWith()
+    {
+        $connection = new Connection(ConnectionManager::config('test'));
+        $targetConnection = new Connection(ConnectionManager::config('test'));
+
+        $this->assertFalse($connection->supportsCrossWith($targetConnection), 'The same connection can\'t used in cross');
```

Comment:
I'm not sure, where do you think it could help? I presume you're talking about `supportsCrossWith`.